content/customize-gnome.md

```diff
@@ -18,117 +18,210 @@ section: software
+</details>
 
-[Sound Output Device Chooser](https://extensions.gnome.org/extension/906/sound-output-device-chooser/)
+### [System76 Power](https://github.com/pop-os/gnome-shell-extension-system76-power)
 
-![sound-output](/images/customize-gnome/sound-output.png)
+<details>
+  <summary>Click to expand</summary>
 
-The Sound Output Chooser adds a quick change option to the system menu in the top right of the screen.  This allows for changing audio inputs and outputs and adjusting volume levels quickly.
+This extension assists with power-management and battery conservation. It adds options for switching CPU performance modes, and enabling/disabling GPU hardware on applicable models.
 
----
+For more information on System76 Power refer to the Github page (above) and/or this [support article](/articles/graphics-switch-pop).
 
-## CPU Power Manager
+</details>
 
-[CPU Power Manager](https://extensions.gnome.org/extension/945/cpu-power-manager/)
+### [Ubuntu AppIndicators](https://github.com/ubuntu/gnome-shell-extension-appindicator)
 
-![cpu-power](/images/customize-gnome/cpu-power.png)
+<details>
+  <summary>Click to expand</summary>
 
-CPU power meter allows for quick switching of various CPU frequency settings.
+The Ubuntu AppIndicators extension enables user-installed applications to display an icon or app-menu in the `top-bar`. For example, in the screenshots below, the **Discord** and **Slack** [icons](/articles/customize-gnome#pop-shell) are present because of this extension.
 
----
+The appearance and features of each icon vary by the application and the developer features enabled.
 
-## Extension Update Notifier
+</details>
 
-[Extension Update Notifier](https://extensions.gnome.org/extension/1166/extension-update-notifier/)
+</details>
 
-This extension will check the various installed extensions and present a notification that there is an update to the extensions.  Extensions are not automatically updated, so this extension provides the next best option.
+## GNOME Tweaks Tool
 
----
+<details>
+  <summary>Click to expand</summary>
+
+Adding and customizing certain features in GNOME desktop environments requires either editing system files or using a tool like GNOME Tweaks.
+
+GNOME Tweaks is available in the **Pop!\_Shop**.
+
+Once installed, users can use GNOME Tweaks (appears as Tweaks after install) to:
+
+- Customize their GNOME theme (icons, colors, etc.).
+- Edit additional mouse and keyboard settings.
+- Change window decorations, such as adding a maximize button, or choosing where app menus appear.
+
+![gnome-tweaks-theming](/images/customize-gnome/gnome-tweaks-themeing.png)
+
+![tweaks-keyboard-mouse](/images/customize-gnome/tweaks-keyboard-mouse.png)
+
+![tweaks-winow-titlebars](/images/customize-gnome/tweaks-window-bars.png)
+
+**NOTE:** GNOME Tweaks is the easiest way to apply installed GNOME themes. However, installing custom themes also requires the [User Themes](#user-themes) extension be installed (see below).
```

Comment:
It would probably be good to make "below" an anchor tag to whatever you're referring to.

Reply:
Fixed.